UltraStar Play/Assets/Scenes/SingingResults/SingingResultsSceneControl.cs

```diff
@@ -201,6 +201,13 @@ private VisualElement GetSelectedLayout()
 
     public void FinishScene()
     {
+        if (sceneData.IsPartyMode)
+        {
+            // TODO intermediary scene that sums up all the rounds after last round
+            SceneNavigator.Instance.LoadScene(PartyModeManager.CurrentRoundData.isLastRound ? EScene.MainScene : EScene.PartyModeVersus);
```

Comment:
- The proper target scene should be selected directly from SingScene instead of redirecting here.